src/components/HelpPanel/HelpPanelContent.tsx

```diff
@@ -1,18 +1,32 @@
 import React from 'react';
 import {
+  Button,
   DrawerActions,
   DrawerCloseButton,
   DrawerHead,
   DrawerPanelBody,
   Title,
 } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
 import HelpPanelCustomTabs from './HelpPanelCustomTabs';
 
 const HelpPanelContent = ({ toggleDrawer }: { toggleDrawer: () => void }) => {
   return (
     <>
       <DrawerHead>
-        <Title headingLevel="h2">Help</Title>
+        <Title headingLevel="h2">
+          Help
+          <Button
+            variant="link"
```

Comment:
Please add `component="a"` and `href=https://status.redhat.com/` for screen readers.
```suggestion
            variant="link"
            component="a"
            href="https://status.redhat.com/"
```